components/component-docs.json

```diff
@@ -6905,7 +6905,7 @@
                     "name": "object"
                 },
                 "required": false,
-                "description": "A custom SVG object to use instead of the supplied SLDS illustrations, look in `design-system-react/assets/images/illustrations` for examples and syntax. _Tested with snapshot testing._ _Tested with Mocha testing._"
+                "description": "A custom SVG object to use instead of the supplied SLDS illustrations, look in `design-system-react/icons` for examples and syntax. _Tested with snapshot testing._ _Tested with Mocha testing._"
```

Comment:
Were the icon SVGs helpful? In what way where they helpful? Are they more helpful than `assets/images/illustrations`?

Reply:
From looking at how it's used (i.e. passed through the custom `Svg` component) it looks like the `illustration` arg is intended to be a custom SVG object, a la `icons/`, not an SVG element. Pointing to `assets/images/illustrations` seems confusing, since passing in an SVG element doesn't work.

Reply:
That's a fair statement.